.circleci/config.yml

```diff
@@ -0,0 +1,215 @@
+# iOS CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/ios-migrating-from-1-2/ for more details
+#
+version: 2
+jobs:
+
+  build:
+    working_directory: ~/code
+    docker:
+      - image: circleci/android:api-27-alpha
```

Comment:
what does the alpha here indicate? Is it some alpha build?

Reply:
Alpha Tag
Below is from CircleCI https://circleci.com/docs/2.0/language-android/#docker-images
Our Android Docker images are currently tagged with the suffix -alpha. This is to indicate the images are currently under development and might change in backwards incompatible ways from week to week